src/braket/pennylane_plugin/translation.py

```diff
@@ -425,6 +436,75 @@ def _(adjoint: Adjoint, parameters):
     return base.adjoint()[0]
 
 
+@_translate_operation.register
+def _(op: ParametrizedEvolution, _parameters, device):
+    start, end = op.t[0], op.t[1]
+    pulse_length = (end - start) * 1e-9  # nanoseconds to seconds
+    pulses = op.H.pulses
+
+    # The driven wires aren't the same as `op.wires` as `op.wires` contains
+    # all device wires due to interaction term.
+    pulse_wires = qml.wires.Wires.all_wires([pulse.wires for pulse in pulses])
+    frames = {w: device.frames[f"q{w}_drive"] for w in pulse_wires}
+
+    # take dt from first frame (all frames have identical dt)
+    time_step = {
+        wire: frame.port.dt * 1e9 for wire, frame in frames.items()
+    }  # seconds to nanoseconds
+
+    pulse_sequence = PulseSequence().barrier(list(frames.values()))
+    callable_index = 0
+
+    for pulse in pulses:
+        # Create waveform for each pulse in `ParametrizedEvolution`
+        if callable(pulse.amplitude):
+            if pulse.amplitude == qml.pulse.constant:
+                amplitude = float(op.parameters[callable_index])
```

Comment:
Can we remove this `float` or set `complex` to allow sending complex constant amplitudes? Below we are also not casting to any specifc `dtype`